### PR TITLE
First pass at binary cleanup

### PR DESCRIPTION
**Pull request by wesboyt**
any input around OO within the binary side would be greatly appreciated!

### CODE REVIEW

**Comment by tgregg** [comment]
Would you mind writing a brief description of the changes you made? I see a lot of whitespace changes, which makes it hard to tell from the diff which changes are consequential.

**Comment by wesboyt** [comment]
it is entirely whitespace/formatting changes and a couple revisions where code was actually broken like:
https://github.com/amzn/ion-js/pull/146/files#diff-daa873b6c5c2943abd7f22cd2fd8d45dL236

File: src/IonBinaryReader.ts [hunk 1]
```diff
@@ -62,194 +62,156 @@ const TB_LIST_CLOSE    = 22;
...
-        s = "$" + symbolId.toString();
-      }
-    }
-    return s;
-  }
+    private _parser: ParserBinaryRaw;
+    private _cat: Catalog;
+    private _symtab: LocalSymbolTable;
+    private _type: number;
+//where do shared symbol tables fall into place here?
```
Comment (tgregg) [inline src/IonBinaryReader.ts]: The reader will encounter shared symbol table imports in the stream and look them up in the provided catalog.

File: src/IonBinaryWriter.ts [hunk 2]
```diff
@@ -58,678 +58,658 @@ enum States {
...
+    private containers: Node[];
+    private fieldName: number[];
+    private state: States;
 
-  writeBlob(value: number[], annotations?: string[]) : void {
-    this.checkWriteValue();
-    if (isNullOrUndefined(value)) {
-      this.writeNull(TypeCodes.BLOB, annotations);
-      return;
+    constructor(symbolTable: LocalSymbolTable, writeable: Writeable) {
```
Comment (tgregg) [inline src/IonBinaryWriter.ts]: Where does this LocalSymbolTable come from?

File: src/IonBinaryWriter.ts [hunk 3]
```diff
@@ -58,678 +58,658 @@ enum States {
...
+        super(writer, parent, TypeCodes.BOOL, annotations);
+        this.value = inputValue;
+    }
+
+    write() : void {
+        this.writeAnnotations();
+        this.writeTypeDescriptorAndLength(this.typeCode, false, this.value ? 1 : 0);
+    }
+
+    getValueLength() : number {//TODO wtf
```
Comment (tgregg) [inline src/IonBinaryWriter.ts]: Boolean values are always encoded in the lower nibble of the type ID byte. So if getValueLength() measures the length past the type ID byte, this is right.

File: src/IonImport.ts [hunk 4]
```diff
@@ -1,85 +1,84 @@
...
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at:
- *
- *     http://aws.amazon.com/apache2.0/
- *
- * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
- * language governing permissions and limitations under the License.
- */
+* Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (tgregg) [inline src/IonImport.ts]: At some point, these need to be updated to 2018. This might not be a bad time since it's mostly whitespace changes anyway.

**Pull request closed without merging**